schemas/edm.xsd

```diff
@@ -628,6 +628,17 @@
       </xs:simpleType>
     </xs:union>
   </xs:simpleType>
+  <xs:simpleType name="TNavigationPropertyType">
+    <xs:union memberTypes="edm:TQualifiedName">

```

Comment:
A qualified name can point to an entity type (allowed here) or to a complex or primitive type (not allowed here). Our XSD cannot distinguish between the two.